provider/pkg/clients/memcache.go

```diff
@@ -1,254 +0,0 @@
-// Copyright 2021, Pulumi Corporation.  All rights reserved.
-
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-//
-// Note: this code is taken nearly verbatim from the upstream memcache client [1].
-// The `refreshLocked` method does not return errors, but instead logs them using
-// the `HandleError` method, which dispatches to klog. This bypasses Pulumi's logging
-// mechanism, and causes unactionable error messages to appear in the CLI.
-// This error logging was disabled.
```

Comment:
Any thoughts on the logging here? Is this still relevant? Would it affect users?

Reply:
Updated. Good catch!